Source/AwsCommonRuntimeKit/mqtt/Mqtt5Client.swift

```diff
@@ -202,11 +197,6 @@ public class Mqtt5Client {
     /// - Throws: CommonRuntimeError.crtError
     public func stop(disconnectPacket: DisconnectPacket? = nil) throws {
         try self.rwlock.read {
-            // validate the client in case close() is called.
```

Comment:
Removed the null validation here. As we are going to handle null client in the aws-c-mqtt: https://github.com/awslabs/aws-c-mqtt/pull/369